src/components/Header.js

```diff
@@ -1,22 +1,43 @@
 import styles from './header.module.css';
-import NavBar from './NavBar'
+import NavLinks from './NavLinks'
+import menu from '../icons/menu.svg'
+import close from '../icons/close.svg'
+import { useState } from 'react';
+
 
 function Header() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const handleOpen = () => {
+    setIsOpen(!isOpen)
+  }
 
-    const navLinks = {
-        "/" : "Home",
-        "/projects" : "Projects",
-        "/contact": "Contact"
-    }
+  const navLinks = [
+    "Home",
+    "Projects",
+    "Contact"
+  ];
 
   return (
-    <div className={styles.header}>
-        <h4><a href='/'>John E Mook</a></h4>
-        <NavBar 
-          navLinks = {navLinks}
-          style = {styles}
+    <header>
+      <div className={styles.header}>
+        <h2><a className={styles.logo} href='#home'>John E Mook</a></h2>
+        <NavLinks
+          navLinks={navLinks}
+          style={styles.nav}
+        />
+        <button onClick={handleOpen} className={styles.navbutton}><img src={isOpen? close: menu}/></button>
+      </div>
+      {isOpen&&
+      <div>
+        <NavLinks
+          navLinks={navLinks}
+          style={isOpen? styles.mobileopen : styles.mobileclosed}
```

Comment:
Can NavLinks handle a class dynamically based on the truthy value of `isOpen`? This would better for css so you could do something like:

```suggestion
          className={'is-open' : isOpen }
```

and then in the css, you'd have

```
.nav-links {
  /* some default styles */

  &.is-open {
    /* styles that will target `.nav-links.is-open` */  
  }
}
```